code/modules/economy/account.dm

```diff
@@ -59,22 +59,32 @@
 /datum/bank_account/proc/add_neetbux()
 	account_job.paycheck += PAYCHECK_WELFARE
 
-/datum/bank_account/proc/bank_card_talk(message)
+/datum/bank_account/proc/bank_card_talk(message, force = 0)
 	if(!message || !bank_cards.len)
 		return
 	for(var/obj/A in bank_cards)
 		var/mob/card_holder = recursive_loc_check(A, /mob)
 		if(ismob(card_holder)) //If on a mob
+			if(card_holder.client)
+				if(!(card_holder.client.prefs.chat_toggles & CHAT_BANKCARD) && !force)
```

Comment:
These checks can be condensed into `if(!(card_holder.client?.prefs.chat_toggles & CHAT_BANKCARD) && !force)`